src/auto/device.rs

```diff
@@ -206,8 +206,7 @@ impl Device {
     }
 
     #[cfg_attr(feature = "v3_20", deprecated)]
-    pub fn grab<'a, P: IsA<Window>, Q: Into<Option<&'a Cursor>>>(&self, window: &P, grab_ownership: GrabOwnership, owner_events: bool, event_mask: EventMask, cursor: Q, time_: u32) -> GrabStatus {
-        let cursor = cursor.into();
+    pub fn grab<P: IsA<Window>>(&self, window: &P, grab_ownership: GrabOwnership, owner_events: bool, event_mask: EventMask, cursor: Option<&Cursor>, time_: u32) -> GrabStatus {
```

Comment:
Should be `IsA<Cursor>` or not?

Reply:
That's automatic generation so if problem there is, it lies in gir.